spec/team_builder_spec.rb

```diff
@@ -62,16 +63,26 @@
                  "govuk-wildcats" => {
                    "channel" => "#govuk-wildcats-team",
                  },
+                 "govuk-generic-team" => {
+                   "channel" => "#govuk-generic-team",
+                   "repos" => [
+                     "generic-repo"
+                   ],
+                 },
                ))
   end
 
   let(:env) { { "SEAL_ORGANISATION" => organisation } }
 
   let(:teams) { TeamBuilder.build(env:) }
 
+  before do
+    allow_any_instance_of(TeamBuilder).to receive(:is_govuk_team?).and_return(false)
```

Comment:
This is [deprecated](https://github.com/rspec/rspec-mocks/issues/1423) - is there another way of getting the test working?